pytorch_trainer/trainer/epoch_based_trainer.py

```diff
@@ -88,6 +88,10 @@ def fit(self, data_loaders, workflow):
                 mode, epochs = flow
                 epoch_trainer = getattr(self, mode)
 
+                # default as -1 means val 1 epoch
+                if epochs == -1:
+                    epochs = 1
```

Comment:
Why does epoch need to set to -1? Any reason?

Reply:
`iter_based_trainer` set to `-1` means run whole train/val set
This make `epoch_based_trainer` has same behavior as `iter_based_trainer`

Reply:
OK